components/ai_chat/resources/page/components/main/style.module.scss

```diff
@@ -44,7 +52,6 @@
 .header {
   padding: 8px 16px;
   border-bottom: 1px solid var(--leo-color-divider-subtle);
-  background-color: var(--leo-color-container-background);
```

Comment:
yes, this is intentional. because the content area should be on top of a container that has a background. this will be evident in the upcoming fullscreen UI feature.